website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, out of the box TypeScript support, Improved Assertions and TODOs
```

Comment:
I'm still not sure about "out of the box TypeScript support" because we do require users to use Babel or `ts-jest`. So it's out of the box only if you use `@babel/preset-typescript`. Maybe:
```suggestion
title: Jest 24: New Website, simpler TypeScript setup, Improved Assertions and TODOs
```